rustworkx/digraph.pyi

```diff
@@ -52,11 +57,20 @@ class PyDiGraph(Generic[S, T]):
         node_map_func: Callable[[S], int] | None = ...,
         edge_map_func: Callable[[T], int] | None = ...,
     ) -> dict[int, int]: ...
+    def contract_nodes(
+        self,
+        nodes: list[int],
```

Comment:
Technically I think this is

```suggestion
        nodes: Sequence[int],
```

although it also will except a `numpy.ndarray`, which isn't a sequence, for all int dtypes I think.